docs/en/ingest-management/data-streams.asciidoc

```diff
@@ -0,0 +1,105 @@
+[[data-streams]]
+= Data streams
+
+{agent} uses data streams to easily store append-only time series data across multiple indices
+while giving you a single named resource for requests.
```

Comment:
```suggestion
{agent} uses data streams to store time series data across multiple indices
while giving you a single named resource for requests.
```

Reply:
My brain tripped over this sentence a couple of times. It's maybe too packed.  Is the term append-only needed?

Reply:
You're right; we don't need that level of detail here.